_tutorials/new-component/create-a-new-component-part-2.md

```diff
@@ -0,0 +1,268 @@
+
+Other naming conventions to keep in mind include:
+- Origami CSS follows the [BEM naming convention](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/).
+- Sass variables are hyphen separated and lowercase.
+- Sass mixins and functions are camel-case.
+- An underscore is used to indicate a [private Sass interface](/spec/v1/sass/#private-sass) which may change at any time and must not be used outside the component.
+
+See the [Sass naming convention part of the specification](/spec/v1/sass/#naming-conventions) for full details and more examples.
+
+## Basic Styles
+
+Let's style our component by adding a border and padding to the `.o-example` CSS class.
+
+<pre><code class="o-syntax-highlight--diff">@mixin oExample ($opts: ()) {
+	// content of primary mixin
+	.o-example {
+-		display: block;
++		border: 1px solid red;
++		padding: 1rem;
++		margin: 0.25rem;
+	}
+}
+</code></pre>
+
+As the demo uses the primary mixin already, refreshing your demo page will show the new styles (provided the `obt dev` command is still running from [part one](/docs/tutorials/create-a-new-component-part-1)).
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/hello-world-demo-2-basic-styles.png" />
+	<figcaption class="o-typography-caption">
+        Our "o-example" "hello world" component demo with a red border, some margin, and some padding.
+	</figcaption>
+</figure>
+
+## Use Other Component Sass
+
+The styles we've added so far use arbitrary colours and spacing (padding/margin) sizes. To improve consistency across projects we can build our component using existing Origami components. To demonstrate lets introduce some existing Origami components to our demo.
+
+First we will update our border colour using [o-colors](https://registry.origami.ft.com/components/o-colors), then we will update our margin/padding to use [o-spacing](https://registry.origami.ft.com/components/o-spacing), we'll style the text content of our component using [o-typography](https://registry.origami.ft.com/components/o-typography), and finally we will add a button using [o-buttons](https://registry.origami.ft.com/components/o-buttons).
+
+### Install Component Dependencies
+
+The first step is to install each component we want to use via [Bower](https://bower.io/). In order for Bower to find the components we will be installing, we need to tell it where to look. For that, we use a `.bowerrc` file. We recommend adding it to your home directory `~/.bowerrc`:
+<pre><code class="o-syntax-highlight--json">{
+	"registry": {
+		"search": [
+			"https://origami-bower-registry.ft.com",
+			"https://registry.bower.io"
+		]
+	}
+}</code></pre>
+
+With Bower configured, we can now run:
+<pre><code class="o-syntax-highlight--bash">bower install o-colors o-spacing o-typography o-buttons --save</code></pre>
```

Comment:
I got to this command and had an error because I haven't got bower installed yet. We can add a prerequisites section and/or make the command use `npx`.

```suggestion
<pre><code class="o-syntax-highlight--bash">npx bower install o-colors o-spacing o-typography o-buttons --save</code></pre>
```

Reply:
I think we may have a future issue here. We are installing the latest version of these components, if they ever release a new major version then we would need to update this tutorial.

Reply:
let's add a prerequisites section 👍 I'll also add major versions to the components (good shout)